### PR TITLE
i18n(zh-cn): Translate `guides/cms/datocms.mdx`

### PR DESCRIPTION
**Pull request by HashCookie**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

Translate `guides/cms/datocms.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+## 与 Astro 集成
+
+在本指南中，你将从 DatoCMS 中获取内容数据，并在 Astro 项目中显示它。
+
+## 前提条件
+
+要开始，你需要准备以下内容：
+
+- **Astro 项目** - 如果你还没有 Astro 项目，请按照我们的[安装指南](/en/install/auto/)中的说明进行设置。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:21]:
```suggestion
- **Astro 项目** - 如果你还没有 Astro 项目，请按照我们的[安装指南](/zh-cn/install/auto/)中的说明进行设置。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+const json = await response.json();
+const data = json.data.home;
+---
+
+<h1>{data.title}</h1>
+
+```
+
+该 GraphQL 查询将从 DatoCMS 项目中获取`home`页面中的`title`字段。刷新浏览器后，你应该在页面上看到标题。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:91]:
```suggestion
该 GraphQL 查询将从 DatoCMS 项目中获取 `home` 页面中的 `title` 字段。刷新浏览器后，你应该在页面上看到标题。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+## 添加动态模块化内容块
+
+如果你的 DatosCMS 项目包含[模块化内容](https://www.datocms.com/docs/content-modelling/modular-content)，那么你需要为每个内容块（例如文本段、视频、引用块等）构建相应的`.astro`组件，以便与项目中的模块字段兼容。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:95]:
```suggestion
如果你的 DatosCMS 项目包含[模块化内容](https://www.datocms.com/docs/content-modelling/modular-content)，那么你需要为每个内容块（例如文本段、视频、引用块等）构建相应的 `.astro` 组件，以便与项目中的模块字段兼容。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+以下示例是用于显示来自 DatoCMS 的"多段文本"块的`<Text />` Astro 组件：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:97]:
```suggestion
以下示例是用于显示来自 DatoCMS 的"多段文本"块的 `<Text />` Astro 组件：
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+const { item } = Astro.props;
+---
+
+<div set:html={item.text} />
+
+```
+要获取这些块，请编辑你的 GraphQL 查询，以包含你在 DatoCMS 中创建的模块化内容块。
+
+在此示例中，DatoCMS中的模块化内容块被命名为**content**。此查询还包括每个项目的唯一`_modelApiKey`，以检查模块字段中的数据，根据内容作者在 DatoCMS 编辑器中选择的内容来动态渲染。在 Astro 模板中使用 switch 语句，根据从查询中接收到的数据进行动态渲染。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:117]:
```suggestion
在此示例中，DatoCMS 中的模块化内容块被命名为 **content**。此查询还包括每个项目的唯一 `_modelApiKey`，以检查模块字段中的数据，根据内容作者在 DatoCMS 编辑器中选择的内容来动态渲染。在 Astro 模板中使用 switch 语句，根据从查询中接收到的数据进行动态渲染。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+以下示例表示了 DatoCMS 模块化内容块，允许作者在主页上选择在文本字段(`<Text />`)和图像(`<Image />`)之间切换：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:119]:
```suggestion
以下示例表示了 DatoCMS 模块化内容块，允许作者在主页上选择在文本字段（`<Text />`）和图像（`<Image />`）之间切换：
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+    }
+  })
+}
+
+```
+
+
+## 发布你的网站
+
+要部署你的网站，请访问我们的[部署指南](/zh-cn/guides/deploy/)并按照适用于你首选的托管提供商的说明进行操作。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:178]:
```suggestion
要部署你的网站，请访问我们的 [部署指南](/zh-cn/guides/deploy/) 并按照适用于你首选的托管提供商的说明进行操作。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+2. 在**Continuous Deployment**选项卡下，找到**Build hooks**部分，然后点击**Add build hook**。
+3. 为 Webhook 提供一个名称，选择要触发构建的分支。点击**Save**并复制生成的URL。
+
+### Vercel
+
+在 Vercel 中设置 Webhook：
+
+1. 进入你的项目仪表板，点击**Settings**。
+2. 在**Git**选项卡下，找到**Deploy Hooks**部分。
+3. 为 Webhook 提供一个名称，选择要触发构建的分支。点击**Add**并复制生成的URL。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:198]:
```suggestion
### Netlify

在 Netlify 中设置 Webhook：

1. 进入你的站点仪表板，点击 **Build & deploy**。
2. 在 **Continuous Deployment** 选项卡下，找到 **Build hooks** 部分，然后点击 **Add build hook**。
3. 为 Webhook 提供一个名称，选择要触发构建的分支。点击 **Save** 并复制生成的 URL。

### Vercel

在 Vercel 中设置 Webhook：

1. 进入你的项目仪表板，点击 **Settings**。
2. 在 **Git** 选项卡下，找到 **Deploy Hooks** 部分。
3. 为 Webhook 提供一个名称，选择要触发构建的分支。点击 **Add** 并复制生成的 URL。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+
+### 添加Webhook到DatoCMS
+
+在你的 DatoCMS 项目管理面板中，导航到**Settings**选项卡，然后点击**Webhooks**。点击加号图标创建一个新的 Webhook，并为其命名。在 URL 字段中，粘贴你所选择的托管服务生成的 URL。作为 Trigger，根据你的需要选择适当的选项（例如：每当新记录被发布时构建）。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:202]:
```suggestion
### 添加 Webhook 到 DatoCMS

在你的 DatoCMS 项目管理面板中，导航到 **Settings** 选项卡，然后点击 **Webhooks**。点击加号图标创建一个新的 Webhook，并为其命名。在 URL 字段中，粘贴你所选择的托管服务生成的 URL。作为 Trigger，根据你的需要选择适当的选项（例如：每当新记录被发布时构建）。
```

File: src/content/docs/zh-cn/guides/cms/datocms.mdx [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+
+## 起始项目
+你还可以在 DatoCMS 市场上查看[Astro博客模板](https://www.datocms.com/marketplace/starters/astro-template-blog)，了解如何使用 Astro 和 DatoCMS 创建博客。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/datocms.mdx:205]:
```suggestion
你还可以在 DatoCMS 市场上查看 [Astro 博客模板](https://www.datocms.com/marketplace/starters/astro-template-blog)，了解如何使用 Astro 和 DatoCMS 创建博客。
```

**Review by 100gle — CHANGES_REQUESTED** [review]

**Review by 100gle — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**